driver/ppm_flag_helpers.h

```diff
@@ -1024,4 +1024,17 @@ static __always_inline u32 unlinkat_flags_to_scap(unsigned long flags)
 	return res;
 }
 
+static __always_inline u32 linkat_flags_to_scap(unsigned long flags)
+{
+	u32 res = 0;
+
+	if (flags & AT_SYMLINK_FOLLOW)
+		res |= PPM_AT_SYMLINK_FOLLOW;
+
+	if (flags & AT_EMPTY_PATH)
```

Comment:
linkat(2) states `AT_EMPTY_PATH` has been added in 2.6.39. If you haven't done it already, it's worth checking if the flag is present under CentOS 6 2.6.32.

Reply:
It's not, I will enclose the check with `ifdef`